src/routes/(auth)/sign-in/+page.svelte

```diff
@@ -26,19 +64,32 @@
 
     <div class="flex flex-col gap-2 mt-6 font-medium">
       <label for="password" class="block">Password</label>
-      <input
-        type="password"
-        id="password"
-        name="password"
-        class="w-full px-4 py-2 border border-black rounded-full"
-        required
-      />
+      <div class="relative">
+        <input
+          type={showPass ? 'text' : 'password'}
+          id="password"
+          name="password"
+          class="w-full px-4 py-2 border border-black rounded-full pr-10"
+          required
+        />
+        <button
+          type="button"
+          class="absolute right-3 top-1/2 transform -translate-y-1/2"
+          on:click={togglePass}
```

Comment:
we could inline this function call since it's just one line.
This works too so this comment is optional.

`on:click={() => showPass = !showPass}`